providers/base/src/clocktest.c

```diff
@@ -36,24 +36,36 @@ int test_clock_jitter(){
         printf("Single CPU detected. No clock jitter testing necessary.\n");
         return 0;
     }
+    else if (num_cpus < 1) {
+        printf("Get number of cpu failed.\n");
```

Comment:
Is this just adding an error trap for a potential issue, or have folks seen cases where this actually happens? (just curious)